Makefile

```diff
@@ -7,8 +7,8 @@
 all: build split-tests
 
 clean:
-	rm -r .build
-	find tests/proofs/ -name '*.k' -delete
+	rm -rf .build/java .build/ocaml .build/logs
+	-find tests/proofs/ -name '*.k' -delete
```

Comment:
Is this change to `find` => `-find` intentional/necessary?

Reply:
yes, find fails if tests/proofs doesn't exist, which it doesn't if you've just checked out the repository, so without this, the make clean target fails.

Reply:
Would `[ ! -d tests/proofs ] || find tests/proofs/ -name '*.k' -delete` be more explicit and easier to understand?

Reply:
I like @nishantjr's suggestion here.